wikimedia/wikipedia-ios
# Image recommendations - Push to commons after tapping image link

**Phabricator:** https://phabricator.wikimedia.org/T358913

### Notes
This PR pushes to the commons URL in a web view after tapping the image link in the bottom sheet.

### Test Steps
1. Open image recommendations, tap image link.
2. Confirm commons page is pushed on in a web view.
